index.js

```diff
@@ -47,6 +49,8 @@ function QuickBooks(consumerKey, consumerSecret, token, tokenSecret, realmId, us
   this.realmId         = eval(prefix + 'realmId')
   this.useSandbox      = eval(prefix + 'useSandbox')
   this.debug           = eval(prefix + 'debug')
+  this.logFunc         = logFunc || console.log
```

Comment:
Should we do anything there to ensure that 'logFunc' is an actual function or is this being too paranoid?

Reply:
good idea